### DISCUSSION TITLE
`lottie-spm`'s Package.swift is not parsable by Dependabot

### DISCUSSION THREAD

**Discussion by hispanico94** [airbnb/lottie-ios#2405] [General]
Hello,

We started incurring in an issue with the integration of Dependabot in our repo where the version updates job would constantly fail. Looking at the logs it seems that the `lottie-spm`'s _Package.swft_ is causing the failure:

```
updater | Computing version for https://github.com/airbnb/lottie-spm.git
updater | error: Invalid manifest (compiled with: ["/opt/swift/usr/bin/swiftc", "-vfsoverlay", "/tmp/TemporaryDirectory.1xSWu9/vfs.yaml", "-L", "/opt/swift/usr/lib/swift/pm/ManifestAPI", "-lPackageDescription", "-Xlinker", "-rpath", "-Xlinker", "/opt/swift/usr/lib/swift/pm/ManifestAPI", "-swift-version", "5", "-I", "/opt/swift/usr/lib/swift/pm/ManifestAPI", "-package-description-version", "5.6.0", "/Package.swift", "-Xfrontend", "-disable-implicit-concurrency-module-import", "-Xfrontend", "-disable-implicit-string-processing-module-import", "-o", "/tmp/TemporaryDirectory.bY8iQv/lottie-spm-manifest"])
updater | error: fatalError
updater | /Package.swift:46:8: error: Linux is currently not supported
updater | #error("Linux is currently not supported")
updater |        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ in https://github.com/airbnb/lottie-spm.git
```

We believe this is caused by the fact that Dependabot runs inside a Linux environment.

We tried adding `lottie-spm` to the list of [ignored dependencies](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#ignore) in our _dependabot.yml_ file, but the problem persists. We found a [comment](https://github.com/dependabot/dependabot-core/issues/8753#issuecomment-1884861510) from a member of the Dependabot team stating that ignored dependencies could still be parsed, which is probably the reason we are still having issues.

We are using `lottie-spm` because of the improved time and space usage compared to `lottie-ios` (and also because of [your recommendation](https://github.com/airbnb/lottie-ios?tab=readme-ov-file#swift-package-manager)).

**Comment by hispanico94** [comment]
Any feedback on this issue?
I'm wondering if that conditional inside the package manifest is still needed

**Reply by rswave** [reply]
any updates on this?

**Comment by AllDmeat** [comment]
Same issue

**Comment by stherold** [comment]
This works now with the lottie upgrade 4.5 🎉
